csharp/DEVELOPER.md

```diff
@@ -36,34 +74,80 @@ source "$HOME/.cargo/env"
 
 Before starting this step, make sure you've installed all software requirments.
 
-1.  Clone the repository:
+1. Clone the repository
+
 ```bash
 VERSION=0.1.0 # You can modify this to other released version or set it to "main" to get the unstable branch
 git clone --branch ${VERSION} https://github.com/aws/glide-for-redis.git
 cd glide-for-redis
 ```
-2.  Initialize git submodule:
+
+2. Initialize git submodule
+
 ```bash
 git submodule update --init --recursive
 ```
-3.  Build the c# wrapper:
-    Choose a build option from the following and run it from the `csharp` folder:
 
-    Build in release mode, stripped from all debug symbols:
+3. Build the C# wrapper
+
+Choose a build option from the following and run it from the `csharp` folder.
+
+Build in release mode, stripped from all debug symbols:
+
+```bash
+dotnet build
+```
+For debug build use:
+```bash
+dotnet build --configuration=Debug
+```
+
+4. Run tests
+
+Run test suite from `csharp` directory:
+
+```bash
+dotnet test
+```
+
+5. Run benchmark
+
+    1. Ensure that you have installed `redis-server` and `redis-cli` on your host. You can find the Redis installation guide at the following link: [Redis Installation Guide](https://redis.io/docs/install/install-redis/install-redis-on-linux/).
```

Comment:
you need `redis-server` for the tests, too.

Reply:
Added, thanks